chain-signatures/contract/src/update.rs

```diff
@@ -35,18 +44,29 @@ pub enum Update {
     Contract(Vec<u8>),
 }
 
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+struct UpdateEntry {
+    updates: Vec<Update>,
+    votes: HashSet<AccountId>,
+    bytes_used: u128,
+}
+
 #[derive(Default, Debug, BorshSerialize, BorshDeserialize)]
 pub struct ProposedUpdates {
-    updates: HashMap<UpdateId, Vec<Update>>,
-    votes: HashMap<UpdateId, HashSet<AccountId>>,
-    next_id: u64,
+    entries: HashMap<UpdateId, UpdateEntry>,
+    id: UpdateId,
```

Comment:
Isn't next_id a better name?

Reply:
It's just an id generator. Calling it next_id meant that it could be sequential when it just generates you an arbitrary id